game-core/src/main/java/games/strategy/triplea/ResourceLoader.java

```diff
@@ -163,8 +137,9 @@ public boolean hasPath(final String path) {
    *     resource. Do not use '\' or File.separator)
    */
   public @Nullable URL getResource(final String inputPath) {
-    final String path = mapPrefix + inputPath;
-    return findResource(path).or(() -> findResource(inputPath)).orElse(null);
+    return findResource(inputPath) //
+        .or(() -> findResource(inputPath))
+        .orElse(null);
```

Comment:
Looks like I missed this in this PR:
findResource() is called twice with the same parameter